documentation/modules/auxiliary/scanner/snmp/snmp_enum.md

```diff
@@ -0,0 +1,78 @@
+## Description
+This module performs a detailed enumeration of a host or a range through SNMP protocol. It supports hardware, software, and network information.
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/snmp/snmp_enum```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```run```
+
+## Scenarios 
+
```

Comment:
```### Negear GSM7224```

Even though there is only one example here, its good to have this so later others can add